### DISCUSSION TITLE
looking for end of support date for rabbit mq client 5.2.0

### DISCUSSION THREAD

**Discussion by Aishbeck** [rabbitmq/rabbitmq-dotnet-client#1692] [Q&A] [answered]
what is the end of support date for rabbit mq client 5.2.0

**Comment by michaelklishin** [comment, marked as answer]
@Aishbeck consider it to be already unsupported. 7.0 is on the verge of shipping, and 6.x is at 6.8.

**Reply by Aishbeck** [reply]
yeah we are aware of it and wil be upgrading in our product next release, but can I get a date /month or year till it was supported?
